modules/fundamental/src/advisory/service/test.rs

```diff
@@ -12,27 +12,34 @@ use trustify_cvss::cvss3::{
 };
 use trustify_module_ingestor::graph::advisory::{
     advisory_vulnerability::{VersionInfo, VersionSpec},
-    AdvisoryInformation,
+    AdvisoryContext, AdvisoryInformation,
 };
+
 use trustify_test_context::TrustifyContext;
 
-async fn ingest_and_link_advisory(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
-    let advisory = ctx
-        .graph
+async fn ingest_sample_advisory<'a>(
+    ctx: &'a TrustifyContext,
+    title: &'a str,
```

Comment:
BTW, not a big deal, but you don't technically need the lifetime on `title`. Because `AdvisoryContext` requires a lifetime, and we're using `ctx` to return one, we're telling the compiler that `ctx` must live at least as long as the value we're returning... if that makes any sense.